third_party/move/move-vm/runtime/src/native_functions.rs

```diff
@@ -164,23 +164,12 @@ impl<'a, 'b, 'c> NativeContext<'a, 'b, 'c> {
         self.resolver.loader().type_to_type_tag(ty)
     }
 
-    pub fn type_to_type_layout(&self, ty: &Type) -> PartialVMResult<Option<MoveTypeLayout>> {
-        match self.resolver.type_to_type_layout(ty) {
-            Ok(ty_layout) => Ok(Some(ty_layout)),
-            Err(e) if e.major_status().status_type() == StatusType::InvariantViolation => Err(e),
```

Comment:
I guess this treatment might be related with the older but now removed feature of native structs. Once those structs where supported, and they would not have a type layout. They are gone, so this change is good.